proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -23,6 +23,16 @@ message HistoricalInfo {
   repeated Validator      valset = 2 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
 }
 
+// Historical contains a set of minimum values needed for evaluating historical validator sets and blocks.
+// It is stored as part of staking module's state, which persists the `n` most
+// recent HistoricalInfo
+// (`n` is set by the staking module's `historical_entries` parameter).
+message Historical {
```

Comment:
Obviously feel free to retain naming here as you like, just wondering why the `Info` suffix was dropped? (apart from the api break etc) 

Personally I find `Historical` alone to be a little ambiguous, would just like to float some suggestions for food for thought:

`History`, `HistoricalEntry`, `HistoricalRecord` or even something similar may play a little better

Reply:
ill go with HistoricalRecord. I didnt want to cause proto breakage and we needed the old type for the migration.

Reply:
Cool, sounds good to me! Thanks